01_background.md

```diff
@@ -55,22 +54,31 @@ This will print the following json response to the terminal:
 }
 ```
 
-Our Rust program will do something very similar. Instead of running `bitcoin-cli decoderawtransaction [raw transaction hex]`, we will call our program using the Rust package manager, Cargo:
+Our Rust program will do something similar.
+Instead of running `bitcoin-cli decoderawtransaction [raw transaction hex]`, we will call our program using the Rust package manager, Cargo:
 
 `
 $ cargo run -- 0100000001c997a5e56e104102fa209c6a852dd90660a20b2d9c352423edce25857fcd3704000000004847304402204e45e16932b8af514961a1d3a1a25fdf3f4f7732e9d624c6c61548ab5fb8cd410220181522ec8eca07de4860a4acdd12909d831cc56cbbac4622082221a8768d1d0901ffffffff0200ca9a3b00000000434104ae1a62fe09c5f51b13905f07f06b99a2f7159b2225f374cd378d71302fa28414e7aab37397f554a7df5f142c21c1b7303b8a0626f1baded5c72a704f7e6cd84cac00286bee0000000043410411db93e1dcdb8a016b49840f8c53bc1eb68a382e97b1482ecad7b148a6909a5cb2e0eaddfb84ccf9744464f82e160bfa9b8b64f9d4c03f999b8643f656b412a3ac00000000
 `
 
 ### Refresher
 
-As a quick refresher, transactions are one of the primary components of Bitcoin. They are a data collection that represent the transfer of money and provide proof to full nodes that a transfer is valid and that the node database and network should be updated to reflect the new ownership. Transaction data is relayed to nodes as a collection of bytes. Nodes and miners, using Bitcoin software, know how to deconstruct and analyze that byte data according to the consensus protocol specification. Chapter 6 of Mastering Bitcoin goes into detail breaking down a transaction. I highly recommend reading this chapter before starting to work on this program. It also offers a useful byte map diagram to visualize the different components:
+Transactions are one of the primary components of Bitcoin.
+A transaction is a data structure that represent the transfer of value in the network.
```

Comment:
Hm I'm not seeing these line breaks in the markdown preview. https://github.com/sb1752/bitcoiner-intro-to-rust/blob/96d5bc488c8b14efe171a418b65750bd5d5bf574/01_background.md

Reply:
That's the idea. Single line breaks in markdown will render as a whole paragraph. Double line breaks start a new paragraph. But single line breaks make small changes in single phrases more evident (for instance, when we change just one word) in the git diffs (see the second commit individually to understand).

Reply:
Ah got it. Makes sense! Thanks